src/cell_group_factory.cpp

```diff
@@ -23,6 +24,9 @@ cell_group_ptr cell_group_factory(const recipe& rec, const group_description& gr
     case cell_kind::lif_neuron:
         return make_cell_group<lif_cell_group>(group.gids, rec);
 
+    case cell_kind::benchmark:
```

Comment:
I wish I could think of a better way to centralize all these switch mappings. New developers will spend a lot of time finding where each mapping happens.

Reply:
We want to make it easy for users to define their own cell/cell_group types.

Some sort of central catalogue of cell types, and associated methods, properties, etc will be required to make that happen.